crud.py

```diff
@@ -0,0 +1,73 @@
+from sqlalchemy.orm import Session
+from typing import List, Optional

```

Comment:
typing module is a built-in module in Python. 

[clarify]: Order of imports: build in packages, external, custom